feat: complete events.Event docs (#35360)
mdn/content: files/en-us/mozilla/add-ons/webextensions/api/events/event/haslistener/index.md

---
title: Event.hasListener()
slug: Mozilla/Add-ons/WebExtensions/API/events/Event/hasListener
page-type: webextension-api-event
browser-compat: webextensions.api.events.Event.hasListener
---

{{AddonSidebar}}

> Check if the listener has been registered to the event.

Reviewer: ```suggestion
Checks if a listener is registered to the event.
```

## Syntax

```js-nolint
events.Event.hasListener(listener)
```

### Parameters

- `listener`

  - : `function`. The listener to check.

### Return value

> A `boolean` value: `true` if the listener has been registered to the event, `false` otherwise.

Reviewer: ```suggestion
A boolean value: `true` if the listener is registered to the event. Otherwise,`false`.
```
